pkg/controller/clusterinstallation/controller.go

```diff
@@ -94,8 +125,11 @@ var _ reconcile.Reconciler = &ReconcileClusterInstallation{}
 type ReconcileClusterInstallation struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver.
-	client client.Client
-	scheme *runtime.Scheme
+	client      client.Client
+	config      *rest.Config
+	podExecutor PodExecutor
+	scheme      *runtime.Scheme
+	state       mattermostv1alpha1.RunningState
```

Comment:
I think `config` is not used anymore, also I believe `state` was removed by @gabrieljackson and is no longer used either.